client/packages/invoices/src/Returns/OutboundDetailView/Footer/StatusChangeButton.tsx

```diff
@@ -124,15 +110,9 @@ const useStatusChangeButton = () => {
   const onConfirmStatusChange = async () => {
     if (!selectedOption || !data) return null;
     try {
-      const status = getStatusUpdateInput(selectedOption.value);
-
-      if (status) {
-        await mutateAsync({
-          outboundReturnId: data?.id,
-          status,
-        });
-        success(t('messages.return-saved'))();
-      }
+      await mutateAsync({ id: data?.id, status: selectedOption.value });
```

Comment:
What does this change do ? Do you mind commenting, i can't see it in the PR description and looks like logic is changed

Reply:
Oh sorry, forgot about this one 🙈 The mapping of InvoiceNodeStatus -> UpdateOutboundReturnStatusInput, I moved it to a parser method in `api.ts`, to align with how we do it elsewhere 😅